trino-rest-github/src/main/java/pl/net/was/rest/github/GithubRest.java

```diff
@@ -925,7 +925,9 @@ public GithubRest(RestConfig config)
     {
         requireNonNull(config, "config is null");
         GithubRest.token = config.getToken();
-
+        if(config.getMinSplits() > 0) {
+            GithubRest.minSplits = config.getMinSplits();
+        }
```

Comment:
If someone tries setting this to 0, it'll be 4 instead, which would be surprising. You can use `com.google.common.base.Verify.verify` here to ensure this value is not 0 and provide an actionable error message:
```suggestion
        verify(config.getMinSplits() > 0, "min_splits must be greater than zero");
        GithubRest.minSplits = config.getMinSplits();
```